Makefile

```diff
@@ -288,11 +288,13 @@ endif
 $(GOPATH)/bin/controller-gen:
 # update this in Nix when upgrading it here
 ifneq ($(USE_NIX), true)
-	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.14.0
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.15.0
 endif
 $(GOPATH)/bin/go-to-protobuf:
 # update this in Nix when upgrading it here
 ifneq ($(USE_NIX), true)
+	# TODO: currently fails on v0.30.3 with
```

Comment:
What package is this?

Reply:
I'm not sure what you mean. It fails to run go install when I bump the version.

```
go install k8s.io/code-generator/cmd/go-to-protobuf@v0.30.3
```